docs/configuration.md

```diff
@@ -1103,10 +1103,10 @@ Apart from these, the following properties are also available, and may be useful
     The number of cores to use on each executor.
 
     In standalone and Mesos coarse-grained modes, setting this
-    parameter allows an application to run multiple executors on the
-    same worker, provided that there are enough cores on that
-    worker. Otherwise, only one executor per application will run on
-    each worker.
+    parameter allows an application to launch multiple executors on the
+    same worker in one single schedule iteration, provided that there are enough
+    cores on that worker. Otherwise, only one executor per application
+    will be scheduled on each worker during one single schedule iteration.
```

Comment:
My hunch is that user may not know the actual meaning of "one single schedule iteration" even if you're trying to clarify the behavior here, unless users know the implementation here.